src/authentication.ts

```diff
@@ -18,6 +20,10 @@ export interface Credentials {
    * The system process id for the LeagueClientUx process
    */
   pid: number
+  /**
+   * Riot Games' self-signed root certificate (contents of .pem)
+   */
+  certificate: string
```

Comment:
This should be optional, we want to use the certificate if the user has it downloaded and provides the file path to it, otherwise we will default to the "unsafe" requests which don't care about SSL validation.